recipe/meta.yaml

```
@@ -190,44 +232,52 @@ outputs:
     script: build-pkg.sh          # [not win]
     script: build-pkg.bat         # [win]
     build:
-      string: "py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ faiss_proc_type }}"                                                  # [cuda_compiler_version == "None"]
-      string: "py{{ CONDA_PY }}cuda{{ cuda_compiler_version|replace(".", "") }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ faiss_proc_type }}"  # [cuda_compiler_version != "None"]
+      # some MKL issue for py 3.8 on osx-64
+      skip: true  # [py==38 and osx and x86_64]
+      string: "py{{ PY_VER|replace(".","") }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ faiss_proc_type }}"                                                  # [cuda_compiler_version == "None"]
+      string: "py{{ PY_VER|replace(".","") }}cuda{{ cuda_compiler_version|replace(".", "") }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ faiss_proc_type }}"  # [cuda_compiler_version != "None"]
     requirements:
       build:
+        - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - {{ compiler('cuda') }}              # [cuda_compiler_version != "None"]
+        - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
         - swig
         - cmake
         - ninja
-        - libgomp                             # [linux]
-        - llvm-openmp                         # [osx]
-        - python                              # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-        - numpy                               # [build_platform != target_platform]
+        # libgomp is required for linux-64 alongside intel-omp.
+        - libgomp                 # [linux]
+        # llvm-openmp is required only for osx
+        - llvm-openmp  14.0.6     # [osx]
       host:
         - python
+        - wheel
+        - packaging
+        - setuptools
         - pip
-        - numpy
+        - numpy  {{ numpy }}
+        # OpenBLAS + libgomp for osx-arm64, and linux-s390x and linux-ppc64le, and linux-aarch64. 
+        - openblas 0.3.21         # [s390x or ppc64le or aarch64 or arm64]     
```

```suggestion
        - openblas {{ openblas }}         # [s390x or ppc64le or aarch64 or arm64]     
```
Because it's inside cbc.yaml https://github.com/AnacondaRecipes/aggregate/blob/346f42e1e793e3617247734a7789ab3238271b2c/conda_build_config.yaml#L163